examples/rms-rt/index.html

```diff
@@ -156,7 +156,7 @@ <h1 class="ui header white-text" style="color: azure;">AudioWorklet example: RMS
       </div>
 
       <div class="ui basic large button" style="margin: 0;">
-        <a href="https://github.com/MTG/essentia.js/tree/master/examples/realtime/audio-worklets" target="_blank" class="ui small button" style="margin: 0;">
+        <a href="https://github.com/MTG/essentia.js/tree/dev/examples/rms-rt" target="_blank" class="ui small button" style="margin: 0;">
```

Comment:
Same comment as above. I suppose these changes has to be made to the other example files as well?